digdag-ui/console.jsx

```diff
@@ -124,6 +127,14 @@ class CodeViewer extends React.Component {
     Ace.acequire('ace/ext/linking')
   }
 
+  getValue () {
+    if (this._editor) {
+      return this._editor.getValue()
+    } else {
```

Comment:
nitpicking: i think the else condition is unnecessary, but i think its a stylistical chioce